app/components/option_item/index.tsx

```diff
@@ -79,12 +93,34 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
 
 const OptionItem = ({
     action, description, destructive, icon,
-    info, label, selected,
+    info, inline = false, label, selected,
     testID = 'optionItem', type, value, containerStyle,
 }: Props) => {
     const theme = useTheme();
     const styles = getStyleSheet(theme);
 
+    const useInLine = useMemo(() => {
+        return inline && Boolean(description);
+    }, [inline, description]);
+
+    const labelStyle = useMemo(() => {
+        return useInLine ? styles.inlineLabel : styles.label;
+    }, [inline, styles, useInLine]);
```

Comment:
inline is not used in the body of the memo, so I do not see any reason to have it in the dependencies. The same for the next two functions.

Reply:
Removed in 5472b2579f694ccb06f222f1ebda965a074feb1f